### PR TITLE
Updating ScrollViewer, FlipView, CalendarView, PipsPager to use carets

### PR DESCRIPTION
**Pull request by beervoley**
## Description

Updating ScrollViewer, FlipView, CalendarView to use carets downlevel.

## Screenshots:
From 191h1

PipsPager (after change):
image
image

PipsPager (before the change):
image
image

ScrollBar (after change):
image
image

ScrollBar (before the change):
image
image

FlipView (after change):
image
image

FlipView (before the change):
image
image

CalendarView (after change):
image

CalendarView (before the change):
image

### CODE REVIEW

File: dev/CommonStyles/Common_themeresources.xaml [hunk 1]
```diff
@@ -47,15 +47,74 @@
...
-    <x:String x:Key="ScrollBarCaretLeft">&#xE0E2;</x:String>
-    <x:String x:Key="ScrollBarCaretRight">&#xE0E3;</x:String>
-    <x:String x:Key="ScrollBarCaretUp">&#xE0E4;</x:String>
-    <x:String x:Key="ScrollBarCaretDown">&#xE0E5;</x:String>
-
-    <x:String x:Key="FlipViewCaretLeft">&#xE0E2;</x:String>
-    <x:String x:Key="FlipViewCaretRight">&#xE0E3;</x:String>
-    <x:String x:Key="FlipViewCaretUp">&#xE0E4;</x:String>
-    <x:String x:Key="FlipViewCaretDown">&#xE0E5;</x:String>
+    <x:String x:Key="LoopingSelectorUpButtonCaretUp">&#xEDDB;</x:String>
```
Comment (licanhua) [inline dev/CommonStyles/Common_themeresources.xaml:50]: remove the two keys and use &#xEDDB and &#xEDDC in style

**Comment by licanhua** [comment]
Can you post the screenshot to compare before and after change?

**Comment by beervoley** [comment]
> Can you post the screenshot to compare before and after change?

will do

**Comment by mdtauk** [comment]
I hope when the CalendarView gets updated those Carets get reduced in size...

**Review by marksfoster — APPROVED** [review]

**Comment by marksfoster** [comment]
Please update FontFamily setters in PipsPager_themeresources.xaml (PipsPagerNavigationButtonBaseStyle and PipsPagerButtonBaseStyle) from StaticResource to ThemeResource:

<Setter Property="FontFamily" Value="{ThemeResource SymbolThemeFontFamily}" />

**Comment by StephenLPeters** [comment]
> Please update FontFamily setters in PipsPager_themeresources.xaml (PipsPagerNavigationButtonBaseStyle and PipsPagerButtonBaseStyle) from StaticResource to ThemeResource:
> 
> `<Setter Property="FontFamily" Value="{ThemeResource SymbolThemeFontFamily}" />`

Why are we doing this?

**Comment by StephenLPeters** [comment]
@beervoley any ideas what might cause this?

**Comment by licanhua** [comment]
> SymbolThemeFontFamily

In OS, SymbolThemeFontFamily may point to MDL2 or Fluent Icons, and we should not hard code it to MDL2

**Review by licanhua — APPROVED** [review]

**Comment by beervoley** [comment]
@licanhua should we hold off merging until everybody agrees on ScrollBar arrow downlevel icon and CalendarView icon sizes?

**Comment by ranjeshj** [comment]
@beervoley can you merge in master ? These files have moved into commonstyles folder.

**Pull request merged**